### PR TITLE
GEODE-7765: Update dependencies for v1.13

### PR DESCRIPTION
**Pull request by metatype**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-lucene/src/integrationTest/java/org/apache/geode/cache/lucene/LuceneIndexCreationPersistenceIntegrationTest.java [hunk 1]
```diff
@@ -228,7 +228,7 @@ private void verifyQueryResultSize(String indexName, String regionName, String q
       String defaultField, int size) throws Exception {
     LuceneQuery query = luceneService.createLuceneQueryFactory().create(indexName, regionName,
         queryString, defaultField);
-    await().untilAsserted(() -> assertThat(query.findPages()).hasSize(size));
+    await().untilAsserted(() -> assertThat(query.findPages()).toIterable().hasSize(size));
```
Comment (kirklund) [inline geode-lucene/src/integrationTest/java/org/apache/geode/cache/lucene/LuceneIndexCreationPersistenceIntegrationTest.java]: I'm surprised this change was needed!

**Comment by metatype** [comment]
I think the `atLeast()` failures are due to https://github.com/awaitility/awaitility/issues/148.

**Comment by metatype** [comment]
Rolling back the jq update for a followup pr, there are extensive API changes.  Also waiting for an awaitility release.

**Comment by onichols-pivotal** [comment]
superseded by #4867

**Pull request closed without merging**